package.json

```diff
@@ -101,7 +101,6 @@
   },
   "dependencies": {
     "@justeat/fozzie": "10.11.1",
-    "@justeattakeaway/pie-icons": "3.2.0",
```

Comment:
Now that pie-icons is defined in the pie-docs/package.json, we don't need to define this dependency anymore.

This was needed because previously, Turborepo didn't know pie-icons was part of the build chain for pie-docs, as it was a pie-monorespo dependency